src/Views/Page.elm

```diff
@@ -88,6 +91,26 @@ stagingAlert { session, loadUrl } =
         text ""
 
 
+newVersionAlert : Config msg -> Html msg
+newVersionAlert { session, reloadPage } =
+    case session.currentVersion of
+        Request.Version.NewerVersion currentVersion newVersion ->
+            div [ class "NewVersionAlert d-block align-items-center" ]
+                [ text "Il y a une nouvelle version de l'application disponible. Vous pouvez la voir en "
+                , a
```

Comment:
I'd prefer a button to better materialize the action is rather impactful. What do you think?